src/Helper/OrganisationApiHelper.php

```diff
@@ -0,0 +1,192 @@
+<?php
+
+namespace Drupal\os2forms_organisation\Helper;
+
+use Symfony\Component\HttpClient\CurlHttpClient;
+
+/**
+ * Organisation API helper class.
+ */
+class OrganisationApiHelper {
+
+  /**
+   * Curl Client.
+   *
+   * @var \Symfony\Component\HttpClient\CurlHttpClient
+   */
+  private ?CurlHttpClient $client = NULL;
+
+  /**
+   * The Settings.
+   *
+   * @var \Drupal\os2forms_organisation\Helper\Settings
+   */
+  private Settings $settings;
+
+  /**
+   * Constructor.
+   */
+  public function __construct(Settings $settings) {
+    $this->settings = $settings;
+  }
+
+  /**
+   * Get curl client.
+   */
+  private function getCurlClient(): CurlHttpClient {
+    if ($this->client) {
+      return $this->client;
+    }
+    else {
+      $this->client = new CurlHttpClient();
+
+      return $this->client;
+    }
+  }
+
+  /**
+   * Get bruger informationer for bruger.
+   *
+   * @phpstan-return array<string, mixed>
+   */
+  public function getBrugerInformationer(string $brugerId): array {
+    $client = $this->getCurlClient();
+
+    $this->settings->getOrganisationApiEndpoint();
+
+    $response = $client->request('GET', $this->settings->getOrganisationApiEndpoint() . 'bruger/' . $brugerId);
+
+    if (200 === $response->getStatusCode()) {
```

Comment:
Is an exception thrown if the status code is not 200?

Reply:
Refactored code. 

The methods calling the API now returns `NULL` if something went wrong and simply the empty string or array if not data is found.

The `NULL` is handled later by showing users a message explaining something went wrong while attempting to fetch organisation data.